libdevice/cmake/modules/SYCLLibdevice.cmake

```diff
@@ -89,13 +96,31 @@ add_custom_command(OUTPUT ${binary_dir}/libsycl-fallback-cassert.spv
                    DEPENDS wrapper.h device.h clang spirv_vars.h llvm-spirv
                    VERBATIM)
 
+add_custom_command(OUTPUT ${binary_dir}/libsycl-fallback-cassert.o
+                   COMMAND ${clang} -fsycl -c
+                           ${compile_opts} ${sycl_targets_opt}
+                           ${CMAKE_CURRENT_SOURCE_DIR}/fallback-cassert.cpp
+                           -o ${binary_dir}/libsycl-fallback-cassert.o
+                   MAIN_DEPENDENCY fallback-cassert.cpp
+                   DEPENDS wrapper.h device.h clang spirv_vars.h clang-offload-bundler
+                   VERBATIM)
+
 add_custom_command(OUTPUT ${binary_dir}/libsycl-fallback-complex.spv
                    COMMAND ${clang} -S -fsycl-device-only -fno-sycl-use-bitcode
                            ${compile_opts}
                            ${CMAKE_CURRENT_SOURCE_DIR}/fallback-complex.cpp
                            -o ${binary_dir}/libsycl-fallback-complex.spv
                    MAIN_DEPENDENCY fallback-complex.cpp
-                   DEPENDS device_math.h device_complex.h device.h clang llvm-spirv
+                   DEPENDS device_math.h device_complex.h device.h clang llvm-spirv clang-offload-bundler
```

Comment:
nit: clang-offload-bundler is not needed here.

Reply:
Thanks, fixed.